### ISSUE TITLE
DevelopmentCodeFragment does not catch fully qualified functions

### ISSUE THREAD

**Issue by bkdotcom** [phpmd/phpmd#846]
- PHPMD version: 2.9.1
- PHP Version:7.4.12
- Installation type: composer
- Operating System / Distribution & Version: OSX 10.14.6

### Current Behavior
`\print_r` not caught

### Expected Behavior
The method foo::bar() calls the typical debug function print_r() which is mostly only used during development.

### Steps To Reproduce:

```
function bar($baz)
{
  \print_r($baz);    // not caught by DevelopmentCodeFragment rule
  print_r($baz);    // caught by DevelopmentCodeFragment rule
}

```
## Checks before submitting
* [x] Be sure that there isn't already an issue about this. See: [Issues list](https://github.com/phpmd/phpmd/issues)
* [x] Be sure that there isn't already a pull request about this. See: [Pull requests](https://github.com/phpmd/phpmd/pulls)
* [x] I have added every step to reproduce the bug.
* [x] If possible I added relevant code examples.
* [x] This issue is about 1 bug and nothing more.
* [x] The issue has a descriptive title. For example:  "JSON rendering failed on Windows for filenames with space".

**Comment by kylekatarnls** [comment]
Confirmed. Actually, this rule doesn't work when there is an active namespace so I actually never had it working in my projects. But allowing starting `\` in function names would be a good start.

**Comment by samuelwang23** [comment]
Could I please be assigned this issue?

**Comment by kylekatarnls** [comment]
With pleasure :) 🥇
